README.md

```diff
@@ -1,24 +1,189 @@
-# README
+<h3 align="center"><b>SpendSmart</b></h3>
 
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure to deploy the project online and add the link in the README file. (You deleted the live demo section from the README file after the last review.)